packages/ffmpeg.rb

```diff
@@ -25,26 +25,22 @@ class Ffmpeg < Package
   depends_on 'fontconfig' # R
   depends_on 'freetype' # R
   depends_on 'fribidi' # R
-  depends_on 'gcc_lib' # R
```

Comment:
Do you get gcc_lib as a dependency if you run the script in an arm container?

Reply:
Ah, I do. hm

Reply:
Fixed.